edx/edx-portal
# UI & dropdown fixes

Fixes/updates the following:
* Does not clear login form on submit (e.g., input stays on error);
* Use `Icon` from Paragon in download CSV button;
* Fixes the position of the download CSV button and last updated date;
* Fixes minor spacing issues throughout;
* Removes `tableSortable` from enterprise list table since sorting is not supported (yet);
* Uses `LinkWrapper` instead of `Link` in `Dropdown` menu items;
* Sets a smaller `max-width` size of 100px on the enterprise logos in the `Header`/`Footer`;
* Upgrades Paragon to latest version;
* Set a height on the loading message for enterprise list;
* Ensure all `button` elements use pointer cursor on hover;
* Adds beta badge back into the `Header`;
* Updates snapshots/tests.
